runtime/src/rpc.rs

```diff
@@ -38,19 +38,19 @@ cfg_if::cfg_if! {
         const DEFAULT_SPEC_NAME: &str = "interbtc-standalone";
         pub const SS58_PREFIX: u16 = 42;
     } else if #[cfg(feature = "parachain-metadata-interlay")] {
-        const DEFAULT_SPEC_VERSION: RangeInclusive<u32> = 3..=3;
+        const DEFAULT_SPEC_VERSION: RangeInclusive<u32> = 3..=4;
         const DEFAULT_SPEC_NAME: &str = "interlay-parachain";
         pub const SS58_PREFIX: u16 = 2032;
     } else if #[cfg(feature = "parachain-metadata-kintsugi")] {
-        const DEFAULT_SPEC_VERSION: RangeInclusive<u32> = 17..=17;
+        const DEFAULT_SPEC_VERSION: RangeInclusive<u32> = 17..=18;
```

Comment:
Are you sure they are backwards compatible?

Reply:
I'm not 100% sure, but we can quickly spin up some nodes on testnet later.